README.md

```diff
@@ -1,4 +1,4 @@
-# Festerize &nbsp;[![Build Status](https://travis-ci.com/UCLALibrary/festerize.svg?branch=main)](https://travis-ci.com/UCLALibrary/festerize)
+# Festerize
```

Comment:
GitHub Actions has a badge too. Here is Fester's for an example:

[![Maven Build](https://github.com/uclalibrary/fester/workflows/Maven%20PR%20Build/badge.svg)](https://github.com/UCLALibrary/fester/actions)

The name of the workflow comes from the name in the YAML file.